pkg/boilerplate/operator/option.go

```diff
@@ -0,0 +1,28 @@
+package operator
```

Comment:
Why do you have a package called `boilerplate`?

Reply:
It is my personal variation on the controller / operator sync loop.  It is used today by the following operators:

1. Alfred
1. Console
1. Authentication
1. Here

My goal was to add it to library-go once I felt it was "good enough."